rattler/templates/tess/index.html

```diff
@@ -0,0 +1,117 @@
+                                        <td>Drittbeste Strategie:</td>
+                                        <td id="strat3"></td>
+                                        <td id="param3"></td>
+                                      </tr>
+                                      <tr>
+                                        <td>Viertbeste Strategie:</td>
+                                        <td id="strat4"></td>
+                                        <td id="param4"></td>
+                                      </tr>
+                                      <tr>
+                                        <td>Fünftbeste Strategie:</td>
+                                        <td id="strat5"></td>
+                                        <td id="param5"></td>
+                                      </tr>
+                                    </tbody>
+                                  </table>
+                            </div>
+                            <div class="col s3" style="margin-bottom: 1%">
+                                <button class="btn waves-effect waves-light" type="submit" id="refreshButton" disabled="true">Berechnen <i class="material-icons right"></i></button>
+                            </div>
+                        </div>
+                        <div class="col s12">
+                            <div class="divider" ></div>
+                        </div>
+                        <div class="row">
+                            <div class="col s3" style="margin-top: 20px">
+                                <b>Beschleunigung des Instruments:</b>
+                            </div>
+                            <div class="input-field col s2" >
+                                <select id="data1">
+                                    <option value="" disabled selected>Wähle deine Spalte</option>
+                                </select>
+                            </div>
+                            <div class="col s1"></div>
+                            <div class="col s3" style="margin-top: 20px">
+                                <b>Existierende Beschleunigung:</b>
+                            </div>
+                            <div class="input-field col s2" >
+                                <select id="data2">
+                                    <option value="" disabled selected>Wähle deine Spalte</option>
+                                </select>
+                            </div>
+                            <div class="col s1"></div>
+                            <div class="col s12" id="validationMessage">
+                                 <h8 style="color:red;">Bitte unterschiedliche Spalten auswählen</h8>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+</form>
+
+{% endblock %}
+{% block sideNavBlock %}
+    {% include 'includeDefs/sideNav.html' %}
+{% endblock %}
+{% block moreScripts %}
```

Comment:
besser mit dem defer_html Modul:
{% load defer %}
{% defer_html %}
<script>
   console.log("abc")
</script>
{% end_defer_html %}

Das muss dann innerhalb eines {%block ...., sonst funktioniert es nicht.

Das ist aber aktuell noch in dem separaten branch "my-project".